phpBB/styles/prosilver/template/ucp_avatar_options.html

```diff
@@ -8,7 +8,7 @@
 	<!-- IF ERROR --><p class="error">{ERROR}</p><!-- ENDIF -->
 		<dl>
 			<dt><label>{L_CURRENT_IMAGE}{L_COLON}</label><br /><span>{L_AVATAR_EXPLAIN}</span></dt>
-			<dd><!-- IF AVATAR -->{AVATAR}<!-- ELSE --><img src="{T_THEME_PATH}/images/no_avatar.gif" alt="" /><!-- ENDIF --></dd>
+			<dd id="avatar-box"><!-- IF AVATAR -->{AVATAR}<!-- ELSE --><img src="{T_THEME_PATH}/images/no_avatar.gif" alt="" /><!-- ENDIF --></dd>
```

Comment:
is this id required somewhere? we should use a class for styling and a data object for js...

Reply:
It's being used but I guess we can replay the ID.